app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum([member.get_rating() for member in team])
+
+
+def elves_concert(elves: list[Elf]) -> list:
+    return [member.play_elf_song() for member in elves]
```

Comment:
according to the task this method should not return anything. just remove the "return" and annotation